### PR TITLE
Exception

### PR DESCRIPTION
**Pull request by Flo-Schield-Bobby**
Depending of the web server configuration, cURL might fail. In my case, curl_errno() returns 51 (CURLE_PEER_FAILED_VERIFICATION)

According to [the libcurl documentation](http://curl.haxx.se/libcurl/c/libcurl-errors.html), it means:
*The remote server's SSL certificate or SSH md5 fingerprint was deemed not OK.*

I added the code corresponding to those exceptions to simplify the debugging.

### CODE REVIEW

File: Ginger/Client/KoalaClient.php [hunk 1]
```diff
@@ -5,6 +5,87 @@ class KoalaClient {
...
+    3 => 'CURLE_URL_MALFORMAT',
+    4 => 'CURLE_URL_MALFORMAT_USER',
+    5 => 'CURLE_COULDNT_RESOLVE_PROXY',
+    6 => 'CURLE_COULDNT_RESOLVE_HOST',
+    7 => 'CURLE_COULDNT_CONNECT',
+    8 => 'CURLE_FTP_WEIRD_SERVER_REPLY',
+    9 => 'CURLE_REMOTE_ACCESS_DENIED',
+    11 => 'CURLE_FTP_WEIRD_PASS_REPLY',
+    13 => 'CURLE_FTP_WEIRD_PASV_REPLY',
+    14 =>'CURLE_FTP_WEIRD_227_FORMAT',
```
Comment (trecouvr) [inline Ginger/Client/KoalaClient.php]: Typo: missing space

File: Ginger/Client/KoalaClient.php [hunk 2]
```diff
@@ -33,21 +114,21 @@ protected function apiCall($endpoint, $params = array(), $method = "GET") {
...
+
     // Si erreur d'appel de cron
     if (curl_errno($ch) != 0) {
-      throw new ApiException(503);
+      throw new ApiException(503, self::$error_codes[curl_errno($ch)]);
     }
     // Si erreur http, on la renvoie
     else if (curl_getinfo($ch, CURLINFO_HTTP_CODE) != 200) {
-      throw new ApiException(curl_getinfo($ch, CURLINFO_HTTP_CODE));
+      throw new ApiException(curl_getinfo($ch, CURLINFO_HTTP_CODE), null);
```
Comment (trecouvr) [inline Ginger/Client/KoalaClient.php]: Passing null explicitly is notre required here

**Comment by trecouvr** [comment]
Thank you for the pull request this is much appreciated.
If you could fix the 2 minor issues I mentioned in my comments I would be able to merge it.

File: Ginger/Client/KoalaClient.php (hunk 2 above)
Comment (Flo-Schield-Bobby) [inline Ginger/Client/KoalaClient.php]: Indeed, I just thought it would be easier for you to see the changes, but I can remove it :)

**Comment by Flo-Schield-Bobby** [comment]
Hey @trecouvr, I just pushed it.

**Comment by trecouvr** [comment]
Thank you for your contribution.

**Pull request merged**